braf/braf.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python3
```

Comment:
e.g. here - could you drop these in any non-executable file?

Reply:
glad to contribute. done.

Reply:
added slight update to imputed_vals dict for knn imputation algo